pkg/scheduler/queue/tree_queue.go

```diff
@@ -141,28 +144,10 @@ func (q *TreeQueue) getOrAddNode(childPath QueuePath) (*TreeQueue, error) {
 	if childQueue, ok = q.childQueueMap[childPath[0]]; !ok {
 		// no child node matches next path segment
 		// create next child before recurring
-		childQueue = NewTreeQueue(childPath[0])
+		// TODO (casie): Need to add support for passing queueingAlgorithms for each child in childPath.
```

Comment:
Still thinking about how to handle this when enqueuing with a path of existing + non-existing nodes.